gravitee-apim-console-webui/jest.config.js

```diff
@@ -1,9 +1,9 @@
 process.env.TZ = 'UTC';
 
 module.exports = {
+  workerIdleMemoryLimit: '896MB',
```

Comment:
📝  for myself : Try to remove this when we use node 20.x 
Deep subject here : https://github.com/jestjs/jest/issues/ 11956